Pod/Classes/PINRemoteImageTask.m

```diff
@@ -83,6 +84,7 @@ - (BOOL)cancelWithUUID:(NSUUID *)UUID manager:(PINRemoteImageManager *)manager
     return noMoreCompletions;
 }
 
+#warning setPriority is not implemented :/
```

Comment:
I think we can leave this without the warning. The subclass download task implements it and calls up to this superclass. Please remove this warning.